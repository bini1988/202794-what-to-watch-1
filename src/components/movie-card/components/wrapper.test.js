import React from "react";
import renderer from "react-test-renderer";
import Wrapper from "./wrapper";

it(`Wrapper correctly renders default markup`, () => {
  const tree = renderer.create(
      <Wrapper className="new-class">
        {`Childs elements`}
      </Wrapper>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
it(`Wrapper correctly renders main markup`, () => {
  const tree = renderer.create(
      <Wrapper className="new-class" main>
        {`Childs elements`}
      </Wrapper>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
