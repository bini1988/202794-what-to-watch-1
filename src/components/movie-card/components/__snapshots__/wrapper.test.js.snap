// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper correctly renders default markup 1`] = `
<div
  className="movie-card__wrap"
>
  Childs elements
</div>
`;

exports[`Wrapper correctly renders main markup 1`] = `
<div
  className="movie-card__hero"
>
  Childs elements
</div>
`;
